docs/designers-developers/developers/block-api/block-registration.md

```diff
@@ -716,6 +716,14 @@ attributes: {
 alignWide: false,
 ```
 
+-   `defaultStyle` (default `true`): When the style picker is shown, a dropdown is displayed so the user can select a default style for this block type. If you prefer not to show the dropdown, set this property to `false`.
```

Comment:
I wonder if there's a risk of ambiguity between `defaultStyle` here and the `isDefault` property of [a registered block variation](https://developer.wordpress.org/block-editor/developers/filters/block-filters/#block-style-variations). I'm not sure I have any suggestion here, but I do note that this particular support is targeted specifically at the control given to the user in selecting their own default, vs. a default amongst a set of registered variations.

Reply:
Maybe `defaultStyleDropdown`, or something of that nature? It's wandering a little to close to tying feature settings to feature implementation details, though. This is particularly evident with this feature, since the first iteration of it was as a set of checkboxes, rather than a dropdown.

Reply:
How about `showDefaultStylePicker`?

Reply:
Since it's a flag in `supports`, the name should read well with that before it. Eg, `defaultStylePicker`, as in "supports default style picker" would potentially work.

Reply:
Renamed in e769c123f.